src/ape/managers/chain.py

```diff
@@ -370,8 +373,14 @@ def pending_timestamp(self) -> int:
         return self.provider.get_block("pending").timestamp
 
     @pending_timestamp.setter
-    def pending_timestamp(self, new_value: int):
-        self.provider.set_timestamp(new_value)
+    def pending_timestamp(self, new_value: str):
+        """
+        Converts date string to timestamp
```

Comment:
Note that doc strs in property setters do not show up.
This is probably better as a comment somewhere.

Reply:
not sure you meant to delete 374 and add those lines, so I didn't include that change
I deleted that docstring and left the #TODO